src/Parser/Hunter/Marksmanship/Modules/Talents/AMurderOfCrows.js

```diff
@@ -37,6 +61,17 @@ class AMurderOfCrows extends Analyzer {
       />
     );
   }
+  suggestions(when) {
+    when(this.shouldHaveSaved).isGreaterThan(0)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(<span>You should generally save <SpellLink id={SPELLS.A_MURDER_OF_CROWS_TALENT_SHARED.id} /> when the boss has under 25% hp so that it is ready to use when the boss hits 20% and you can start getting <SpellLink id={SPELLS.BULLSEYE_TRAIT.id}/> quicker.</span>)
```

Comment:
"Don't cast between 20 and 25%" is a strangely specific suggestion, I hope that it's relevant as you think. I can see from the descriptions of Bullseye and Murder of Crows that this probably makes sense, tho.

Reply:
Yeah it is very specific - but it's a general rule of thumb, that helps the player quickly rack up bullseye stacks which is a large part of our damage. (Each tick adds a stack for a total of 16 stacks over 15 seconds (or until 30 in total)). 

I'll degrade it to be average importance and emphasize generally.

Reply:
Ehh you don't have to change it, was just making sure it needed to be that specific